src/renderers/testing/__tests__/ReactTestRenderer-test.js

```diff
@@ -231,6 +232,84 @@ describe('ReactTestRenderer', () => {
     );
   });
 
+  it('allows an optional getMockRef function', () => {
```

Comment:
The way "options" are generally handled in JS libraries, user-passed options are shallowly merged with defaults, and the result is passed down. So passing `null`, `{}` or `{unrelatedOption:42}` gives you default `getMockRef` implementation.

Can we do the same here? `options = Object.assign({}, options, defaultOptions)`